lib/active_record/virtual_attributes/virtual_fields.rb

```diff
@@ -115,39 +115,7 @@ def preloaders_for_reflection(reflection, records, scope, polymorphic_parent)
               super(reflection, records, scope)
             end
           end
-        else
```

Comment:
the `if` on line 104 isn't needed now right?

Reply:
✂️ 🔥 ✂️ 🔥 ✂️ 🔥 

🍰  🍪 👏 🙇 😍 🎉

Reply:
Oh yeah...is this PR broken then?

Reply:
Ok deleted now :)

Reply:
yea. this is the indentation I was mentioning. I fixed.

Hmm. The else and the end and the if went away since those were grouped.